package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-plugin-react-redux",
   "description": "Integrates React and Redux to a Fusion.js application",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
You can also remove this change as we will bump the version through our release process